sdks/wasm/DebuggerTestSuite/Tests.cs

```diff
@@ -1243,6 +1247,141 @@ public async Task InspectLocalsForStructInstanceMethod (bool use_cfo)
 							label: "this#0");
 
 				});
+
+		[Fact]
+		public async Task SetBreakpointBeforeInspectorReady () {
+			var insp = new Inspector ();
+			
+			var scripts = SubscribeToScripts (insp);
+			await Ready (); 
+
+			using (var cts = new CancellationTokenSource()) {
```

Comment:
Duplicating the code here feels brittle, because this could potentially become ineffective if there are changes in `insp.Ready` that we are needed here too, but don't get copied over. @lewing ?